ignition/service/requestqueue.py

```diff
@@ -207,6 +209,7 @@ def handle_request(self, request):
             request_as_dict['system_properties'] = PropValueMap(request_as_dict['system_properties'])
             request_as_dict['request_properties'] = PropValueMap(request_as_dict['request_properties'])
             request_as_dict['associated_topology'] = AssociatedTopology.from_dict(request_as_dict['associated_topology'])
+            request_as_dict['version'] = REQUEST_MESSAGE_VERSION
```

Comment:
I don't understand why the version needs to be set here during message consumption from Kafka (it should be set only on message delivery, shouldn't it)?

Reply:
Ok, I understand. It is in request_as_dict already, so no need for this line as Dan states.

Reply:
Addressed in commit 5.